xbmc/video/VideoDatabase.h

```diff
@@ -67,6 +67,18 @@ namespace VIDEO
   struct SScanSettings;
 }
 
+enum EDetailsFlag
```

Comment:
@Montellese  With class you have to define bitmask operators to make it work.. Is it worth it or maybe just going with a better naming for it?